README.md

```diff
@@ -51,29 +52,31 @@ The following example shows the format of the text file containing frame filenam
 ./anim1/frame04.png 600
 ```
 
-**WebP Encoding Options:**
+**Options:**
 
-| Option | Default Value | Description|
-|--------|:-------------:|------------|
+| <img width="150"> Option | Default Value | <img width="600"> Description|
+|--------------------------|:-------------:|------------------------------|
 |`-soft_max_size`|153600|Desired (soft) maximum size limit (in bytes).|
 |`-hard_max_size`|153600|Hard limit for maximum file size (in bytes).|
 |`-loop_count`|0 infinite loop|Number of times animation will loop.|
 |`-min_lossy_quality`|0|Minimum lossy quality (0..100) to be used for encoding each frame.|
 |`-m`|4|Effort/speed trade-off (0=fast, 6=slower-better).|
 |`-allow_mixed`|false|Use mixed lossy/lossless compression.|
-
-**Thumbnailer Algorithm Options:**
-
-| Option | Default Value | Description|
-|--------|:-------------:|------------|
-|`-equal_quality`|true|Generate animation setting the same quality to all frames.|
-|`-equal_psnr`|false|Generate animation so that all frames have the same PSNR.|
-|`-near_ll_diff`|false|Generate animation allowing near-lossless method, impose different pre-processing to near-losslessly-encoded frames.|
-|`-near_ll_equal`|false|Generate animation allowing near-lossless method, impose the same pre-processing to near-losslessly-encoded frames.|
-|`-slope_optim`|false|Generate animation with slope optimization.|
+|`-algorithm`|equal_quality|Algorithm to generate animation {equal_quality, equal_psnr, near_ll_diff, near_ll_equal, slope_optim}.|
 |`-slope_dpsnr`|1.0|Maximum PSNR change (in dB) used in slope optimization.|
 |`-verbose`|false|Print various encoding statistics showed by chosen algorithm.|
 
+**`-algorithm` flag description:**
```

Comment:
You should use `## Title` here I think. See https://google.github.io/styleguide/docguide/style.html